File: Android-Project/app/src/main/java/sample/jchankchan/myapp/MainActivity.java
package sample.jchankchan.myapp;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import sample.jchankchan.lib.Fuga;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public static String methodForJunit() {
        return Fuga.methodForJunit();
    }

    public String methodForEspresso() {
        return new Fuga().methodForEspresso();
    }
}

File: Android-Project/kotlinapp/src/main/java/sample/jchankchan/kotlinapp/MainActivity.kt
package sample.jchankchan.kotlinapp

import android.os.Bundle
import androidx.annotation.VisibleForTesting
import androidx.appcompat.app.AppCompatActivity
import sample.jchankchan.lib.FooBar

class MainActivity : AppCompatActivity() {

    @VisibleForTesting
    var fooBar = FooBar()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun methodFromKotlinEspresso(): String? {
        return fooBar.methodFromKotlinEspresso()
    }

    companion object {
        fun methodFromKotlinJunit(): String? {
            return FooBar.methodFromKotlinJunit()
        }
    }
}